shared/chat/conversation/messages/coinflip/container.js

```diff
@@ -36,6 +36,9 @@ const mapStateToProps = (state, {flipGameID}: OwnProps) => {
 export default namedConnect<OwnProps, _, _, _, _>(
   mapStateToProps,
   (d, o) => ({}),
-  (s, d, o) => ({...s}),
+  (s, d, o) => ({
+    ...s,
+    howThisWorksURL: 'https://keybase.io/coin-flip',
```

Comment:
i don't think this needs to be in the container, just move it into the components and itd remove a bunch of code

Reply:
👌🏻